tests/pages/components/slugged-route-model-details.js

```diff
@@ -0,0 +1,11 @@
+
+import organizationProfile from 'code-corps-ember/tests/pages/components/organization-profile';
+import userDetails from 'code-corps-ember/tests/pages/components/user-details';
+
+export default ({
+  scope: '.slugged-route-model-details',
+
+  organizationProfile,
+  userDetails
+
```

Comment:
Please watch the spacing here. There is no need for a line break here or on line 1. Don't let that become a recurring problem where it's habitual.